Description: Fix SEGV printing PostScript in latin-1 locale
Author: Olly Betts <olly@survex.com>
Bug-Debian: http://bugs.debian.org/702378
Bug: http://trac.wxwidgets.org/ticket/15489
Forwarded: http://trac.wxwidgets.org/ticket/15489
Last-Update: 2013-09-11

--- a/src/generic/dcpsg.cpp
+++ b/src/generic/dcpsg.cpp
@@ -2340,8 +2340,20 @@
 
     long sum=0;
     float height=fontSize; /* by default */
-    unsigned char *p;
-    for(p=(unsigned char *)wxMBSTRINGCAST strbuf; *p; p++)
+    unsigned char *p=(unsigned char *)wxMBSTRINGCAST strbuf;
+    if(!p)
+    {
+        // String couldn't be converted which used to SEGV as reported here:
+        // http://bugs.debian.org/702378
+        // http://trac.wxwidgets.org/ticket/15300
+        // Upstream suggests "just return if the conversion failed".
+        if (x) (*x) = 0;
+        if (y) (*y) = 0;
+        if (descent) (*descent) = 0;
+        if (externalLeading) (*externalLeading) = 0;
+        return;
+    }
+    for(; *p; p++)
     {
         if(lastWidths[*p]== INT_MIN)
         {
