Description: Fix code typo
 = should be ==!
Author: Olly Betts <olly@survex.com>
Last-Update: 2013-09-10
Bug: http://trac.wxwidgets.org/ticket/15490
Forwarded: http://trac.wxwidgets.org/attachment/ticket/15490/wxpython-%3D-should-be-%3D%3D.patch

--- a/wxPython/src/_bitmap.i
+++ b/wxPython/src/_bitmap.i
@@ -308,7 +308,7 @@
                         p.OffsetY(pixData, 1);
                     }
                 }
-                if (depth = 32) {
+                if (depth == 32) {
                     MAKE_PIXDATA(wxAlphaPixelData);
                     for (int y=0; y<height; y++) {
                         rowStart = p;
--- a/wxPython/src/gtk/_gdi_wrap.cpp
+++ b/wxPython/src/gtk/_gdi_wrap.cpp
@@ -3277,7 +3277,7 @@
                         p.OffsetY(pixData, 1);
                     }
                 }
-                if (depth = 32) {
+                if (depth == 32) {
                     MAKE_PIXDATA(wxAlphaPixelData);
                     for (int y=0; y<height; y++) {
                         rowStart = p;
